src/helpers/hex_grid/axial.rs

```diff
@@ -35,8 +35,8 @@ pub struct AxialPos {
     pub r: i32,
 }
 
-impl From<&TilePos> for AxialPos {
-    fn from(tile_pos: &TilePos) -> Self {
+impl From<TilePos> for AxialPos {
```

Comment:
I have heard that doing `From<AsRef<TilePos>` allows us to retain flexibility of calling `from` on a reference *or* value of something. Is this true? In this case, that might be a good option here?

Reply:
Yeah, I think that could be nicer. Maybe I'll play a bit more around with that and maybe split this into separate prs